webpack.config.js

```diff
@@ -39,6 +45,24 @@ module.exports = env => {
         {
           test: /\.html?$/,
           use: [{ loader: 'html-loader' }]
+        },
+        {
+          test: /\.(css|scss|sass)$/i,
+          use: [
+            {
+              loader: 'style-loader'
```

Comment:
Let's use this: https://github.com/webpack-contrib/mini-css-extract-plugin#configuration. It will allow us to code split css later. We will also want to look at minimizing for production too: https://github.com/webpack-contrib/mini-css-extract-plugin#minimizing-for-production, but that can wait for later as well since I believe Patternfly CSS is already minified.

Reply:
This comment go down ^^. I think that there is a min version for patternfly.